### DISCUSSION TITLE
Job Search Sites

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#894] [Job Resources]
If you have a job search site, add them below.

**Comment by BekahHW** [comment]
## Remote-friendly

- https://otta.com/
-  https://www.toptal.com/ (they have a test)
-  https://weworkremotely.com/
-  https://remoteok.com/
-  https://www.workingnomads.com/jobs
-  https://remotive.com/

**Reply by Bolaji5821** [reply]
Stack Overflow Jobs: Stack Overflow is a Q&A platform for programmers, and they also have a job section where employers can post their job vacancies. The platform allows you to filter by 'Remote' to find jobs that offer remote working options.

Remote OK: This website is specifically focused on remote jobs in the tech sector. You can find many listings for software engineering roles from all over the world.

We Work Remotely: This is another site dedicated solely to remote job listings. They have a tech-focused job board that's frequently updated with new positions.

FlexJobs: While this is a paid service, FlexJobs curates a list of verified, scam-free remote jobs. This can save you a lot of time and hassle in the job search process.

GitHub Jobs: Given that GitHub is a platform for software developers, its job board often lists quality software engineering positions, many of which offer remote options.

AngelList: Aimed at startups, AngelList allows you to look for jobs in early-stage companies. There’s an option to filter by remote roles.

Remote .co: This platform specializes in remote job listings, and its section for software development often has a good selection of available roles.

LinkedIn: Over the years, LinkedIn has seen an increasing number of remote job listings. You can use the 'Jobs' tab and filter by location to see "Remote" job offerings.

Dribbble Jobs: Though Dribbble is primarily known as a platform for designers, their job board also has listings for developer jobs, many of which are remote.

Remotive: Remotive is another job board that focuses solely on remote positions, and they frequently update their software development category.

Working Nomads: This site curates a list of remote job offerings from various digital professions, including software engineering.

Jobspresso: This platform is focused on listing remote jobs in tech, marketing, and customer support. The software engineering section frequently has new listings.

Remotees: Remotees aggregates remote job listings from several of the previously mentioned websites, making it a good platform to get a snapshot of many listings at once.

JustRemote: Focused purely on remote job listings, JustRemote covers a range of job categories including software development.

Remote Tech Work: As the name suggests, this site is dedicated to tech-related remote job listings, making it perfect for software engineers.

Remote Python: If you're specifically a Python developer, this site is a niche platform with remote job listings for Python roles.

Authentic Jobs: This platform has both design and developer jobs, many of which are remote. It's been around for quite some time and has a reputation for quality listings.

RubyNow: For Ruby developers, RubyNow is a niche job board focused on Ruby and Ruby on Rails positions. Many of the listings are remote.

Remote Job Hub: This is an aggregator that pulls in remote job listings from other boards. It has a tech category where you can find software engineering roles.

Remote People: This platform lists jobs in a variety of categories, including software engineering and development.

Europe Remotely: If you're looking to work remotely but with companies based in European time zones, this is a great niche site.

SkipTheDrive: A platform dedicated to telecommuting jobs across multiple sectors, including technology.

Remote4Me: An aggregator for remote tech jobs. It’s handy since it crawls multiple job boards and company websites to gather listings in one place.

Vue Jobs: If you specialize in Vue.js, this niche job board might be ideal for you. Many of the positions listed offer remote options.

Landing .jobs: While it has a focus on European tech job markets, there are numerous remote opportunities listed.

CyberCoders: This is a recruiting firm that frequently has listings for remote tech positions.

Remote Circle: A job board that allows you to search for jobs available in your time zone or any time zone you prefer.

Who is Hiring: It’s a map interface that shows tech jobs around the world, and you can filter by those that are remote.

NoDesk: While not solely focused on tech jobs, NoDesk has a remote job category dedicated to software development roles.

Pangian: A platform dedicated to connecting remote workers with opportunities globally. The tech section often features software development roles.

Daily Remote: Offers daily listings of remote jobs in various categories, including software development.

Remote Work Hub: Another platform dedicated to remote-only job listings across various categories, including tech.

GoRemote: A simple, straightforward site that aggregates remote job listings, many of which are in the tech sector.

StartUs: This platform is known for startup jobs, and many startups offer remote positions for developers.

Find Bacon: A job board that caters mainly to design and development jobs, with a fair number of them being remote.

Remote Developers: As the name suggests, it’s a niche site for remote developer jobs.

PowerToFly: Initially started to connect women with tech jobs, this platform now offers diverse job listings, including many remote tech positions.

TechMeAbroad: Focuses on tech jobs that sponsor visas for those looking to work abroad, but they also have listings for companies offering remote roles.

Remotely Awesome Jobs: An aggregator of remote jobs from various other job boards. While it covers many sectors, the tech section is quite comprehensive.

Jobscribe: A service that sends daily emails with the latest tech startup jobs, many of which are remote positions.

Y Combinator Jobs: Y Combinator's own job board often has listings from startups, and a significant number offer remote roles.

JS Remotely: Specifically for JavaScript developers, this board lists remote jobs centered around JavaScript.

Workew: A platform that lists a variety of positions, including those in tech, from companies that are open to hiring remote workers.

Remote Jobr: A job board that aggregates listings from various sources and offers a section dedicated to developer roles.

WFH Jobs: "Work From Home" job board includes a variety of listings, and the tech section is particularly active.

Remote Leads: A service that sends remote job listings directly to your inbox, many of which are tech-related.

Remote Planet: A broader platform for all things remote work-related, including a job board with tech positions.

RemoteHub: Lists remote jobs alongside company cultures, giving you insights into the working environment of potential employers.

Remotists: This platform curates a newsletter that sends weekly lists of remote jobs in various sectors, including technology.

Remote Bliss: A platform focusing on remote and work-from-home jobs in various sectors, including technology.

Remote Global: Features remote job listings from around the world across various sectors.

Hubstaff Talent: Hubstaff’s platform for remote jobs has a variety of tech-related listings.

WorkRemote: A straightforward site that lists remote job opportunities in various sectors, including tech.

Remote Design Jobs: While the main focus is on design, there are often tech and developer roles given the intertwining nature of design and development in many projects.

Remote Bond: Showcases a variety of remote positions, from full-time roles to freelance opportunities, including in the tech sector.

Remote Club: This platform features a variety of remote listings, with a section dedicated to development jobs.

Remote OK Jobs: Not to be confused with the other "Remote OK" mentioned earlier, this platform has a wide variety of remote listings, including plenty in the tech domain.

Time Doctor Remote Job Board: Time Doctor, known for its productivity software, also has a job board that frequently lists remote tech positions.

**Comment by rohan057** [comment]
Hi guys we have a Graduate Programme for 2024 if interested read below and apply.

Graduate Programme 2024: Software Development

About Us:
Accolite Digital, a leading digital transformation services provider, is excited to announce the launch of its Software Development Graduate Programme in 2024.

Position: Software Developer - Python/Java

Location: Glasgow

Programme Start: January 2024

Your Role:
As a Software Developer with Accolite Digital, you will have the opportunity to work on-site, starting in January 2024. Your initial engagement will be with a Fortune 100 global investment bank in the financial services industry. Responsibilities include designing, developing, modifying, and maintaining software products.

What We Offer:

Comprehensive development plan
Extensive and continuous training in banking, financial services, and relevant technologies
Exposure to cutting-edge technology and a dynamic community of technologists
Requirements:

MSc/BSc in a technical-related degree (Computer Science, Computing-Related Fields)
Graduated between 2019-2022
Genuine interest in technology and programming
Coding proficiency in Java and/or Python
Strong understanding of object-oriented concepts
Attention to detail, quality-focused, and adherence to standards/best practices
Ability to thrive in a fast-paced environment
Excellent written and verbal communication skills
About Accolite Digital:
Accolite Digital is an innovative digital transformation services provider, applying design thinking to solve complex business problems for Fortune 500 companies. We believe in a bias-free world and encourage diversity in our workforce.

Why Join Us:
At Accolite Digital, we believe a bias-free world is an empowered world. We welcome applications from everyone, especially women and individuals from diverse backgrounds, to contribute to building the next generation of Accolite Digital.

Ready to Launch Your Career in Software Development?
Apply now for the Software Development Graduate Programme and be part of a community of bright and passionate technologists shaping the future at Accolite Digital.

Python Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37
Java Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37

Reply back if you have any further questions and I will be happy to answer, else best of luck with your applications.
